source/features/table-input.tsx

```diff
@@ -70,6 +69,13 @@ function init(): void {
 	}
 }
 
+function init(): void {
+	delegate(document, '.rgh-table-input-cell', 'click', addTable);
+	delegate(document, '.rgh-table-input-cell', 'mouseenter', highlightSquares, {capture: true});
```

Comment:
I'm still disappointed we need this delegated mousenter event because it triggers hundreds of times for every element when the mouse just moves around. Delegate then uses element.match() to avoid calling this callback. 

I remember we tried a CSS-only solution but it didn't work maybe. Can you think of anything else? How often would an onmouseenter inline listener break?

Reply:
I'll try something in a separate PR.